aries-site/src/pages/templates/forms.mdx

```diff
@@ -23,29 +23,67 @@ import { Example } from '../../layouts';
   <SimpleSignUpExample />
 </Example>
 
-## Guidance
+### Jump to section...
```

Comment:
Maybe we should omit the "Jump to section" on this PR since we don't have a design direction and leave that to be handled by separate intern project work.

Reply:
It might take a while for Aditi and I to develop a standardized look for the table of contents (i.e. Jump to section). In the meantime I would really want to see what the reaction is to at least the concept.